nautobot/extras/templatetags/job_buttons.py

```diff
@@ -69,115 +70,108 @@
 </div>
 """
 
+SAFE_EMPTY = mark_safe("")  # noqa: S308
```

Comment:
```suggestion
SAFE_EMPTY_STR = mark_safe("")  # noqa: S308
```
Make it easier to determine what this variable is doing without context later in the file.